duckduckgo/iOS
# Promote autofill features to remotely trigger-able

Task/Issue URL: https://app.asana.com/0/72649045549333/1203676877763836/f
Tech Design URL: https://app.asana.com/0/481882893211075/1204144061793469/f
CC:

**Description**:

On the back of the changes made in https://github.com/duckduckgo/iOS/pull/1555 to introduce a FeatureFlagger + autofill Privacy-Config subfeatures to BSK, this PR adds support for the newly added internal case of the state enum field of both the top level features and their nested features.

This change will also allow us to release changes that will only be visible to internal users, then enable them for end users remotely which we will use for the autofill release.



**Steps to test this PR**:
1. Check that any features configured using the privacy-config still work as expected (particularly autofill).
2. Check feature flags (autofill, debug menu, sync)
3. Test credentials autofill



**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
